src/libpriv/rpmostree-kernel.cxx

```diff
@@ -627,16 +626,10 @@ rpmostree_run_dracut (int     rootfs_dfd,
    * https://bugzilla.redhat.com/show_bug.cgi?id=1401444
    * https://bugzilla.redhat.com/show_bug.cgi?id=1380866
    * */
-  random_cpio_data = g_resources_lookup_data ("/rpmostree/dracut-random.cpio.gz",
-                                              G_RESOURCE_LOOKUP_FLAGS_NONE,
-                                              error);
-  if (!random_cpio_data)
-    return FALSE;
-  gsize random_cpio_data_len = 0;
-  auto random_cpio_data_p = static_cast<const guint8*>(g_bytes_get_data (random_cpio_data, &random_cpio_data_len));
+  auto random_cpio_data = rpmostreecxx::get_dracut_random_cpio ();
   if (lseek (tmpf.fd, 0, SEEK_END) < 0)
     return glnx_throw_errno_prefix (error, "lseek");
-  if (glnx_loop_write (tmpf.fd, random_cpio_data_p, random_cpio_data_len) < 0)
+  if (glnx_loop_write (tmpf.fd, random_cpio_data.data(), random_cpio_data.length()) < 0)
```

Comment:
Also I know the parenthesis spacing is inconsistent.  It's a challenge mentally for me at least because the methods being shared with Rust pulls towards Rust style.  And in particular for method invocations there's a strong "binding" between the identifier and its arguments.  Most C++ projects don't use a space between method name and open paren; there isn't an offical GNU coding style for C++ AFAICS.

Overall this kind of fits in with a theme of having the C/C++ get a bit uglier in some cases in return for making it *way* easier to bridge to Rust.

Also, perhaps at some point we can investigate using [clang-format](https://clang.llvm.org/docs/ClangFormat.html).